locales/index.js

```diff
@@ -6,6 +6,16 @@ function getLocales(arr) {
 
 var reqF = require('enhanced-require');
 var req = reqF(module);
-var arr = !req.context ? require.context('./', true, /\.json$/).keys() : req.context('./', true, /\.json$/).keys();
+if (!req.context) {
+  req = require;
+}
```

Comment:
The above if block isn't required for the locale building, I just thought it was clearer way to normalize a usable req.context, instead of the previous ?: operator.